### DISCUSSION TITLE
USB audio input

### DISCUSSION THREAD

**Discussion by digitalelements** [tttapa/Control-Surface#843] [Q&A]
Hi There,

I noticed in the information about Control-Surface that it is possible to create USB audio output.
Is it also possible to create audio inputs ? I would be using a Teensy 4. does this mean the Teensy can become a class compliant audio interface ?

Kind Regards,

Chris

**Comment by tttapa** [comment]
Yes, you can use the Teensy Audio library: https://www.pjrc.com/teensy/td_libs_Audio.html

**Reply by digitalelements** [reply]
Hi,

Thanks so much for the quick reply. I've looked over the ASD sometime ago. But, being so new was unable to ascertain what was needed to allow the teensy to be seen as a class compliant audio interface. I will dive in again to explore. any advice or clues would be welcome and greatly appreciated.

Regards,

Chris
